scripts/release.js

```diff
@@ -156,6 +156,21 @@ logger.warn(`
 
 shouldContinuePrompt();
 
+// Creates a new React test app, install component and sdk and build the test app to ensure no peer dependency warnings, errors or build issues
+logger.log('Create a new npm package and installl component and js sdk as dependencies, check if there is peer dependency warning for component library');
```

Comment:
nit: install typo

Reply:
Good catch. Corrected in the lasted commit.